core/domain/state_domain.py

```diff
@@ -34,6 +34,9 @@
 from core.domain import translation_domain
 from extensions.objects.models import objects
 
+from typing import Any, Dict, List, Optional, Union
```

Comment:
Add explanation for `Any` in state domain

Reply:
You should probably try to remove it altogether and have the correct answer be a union type, actually. "Any" literally means any.

Reply:
Any is removed for `Rulespec`